### PR TITLE
Set platform-variable in windows

### PR DESCRIPTION
**Pull request by rodolfocarobene**
Hello!
I was trying to prepare a windows environment to use Qibolab and I noticed that the `export` command behaves a bit differently there.

I changed the tutorial accordingly. 

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [ ] Tests are passing.
- [ ] Coverage does not decrease.
- [ ] Documentation is updated.

### CODE REVIEW

**Review by andrea-pasquale — APPROVED** [review]
Thanks @rodolfocarobene.

**Review by stavros11 — APPROVED** [review]
Cannot test on windows but I believe you.

**Comment by andrea-pasquale** [comment]
> Cannot test on windows but I believe you.

I can vouch for him given that I saw the fix working IRL :)

File: doc/source/getting-started/experiment.rst [hunk 1]
```diff
@@ -107,12 +107,19 @@ Setting up the environment
 --------------------------
 
 After defining the platform, we must instruct ``qibolab`` of the location of the create file.
-This can be done using an environment variable:
+This can be done using an environment variable.
```
Comment (andrea-pasquale) [inline doc/source/getting-started/experiment.rst:110]:
```suggestion
This can be done using an environment variable:
```

**Pull request merged**